ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/commitmarkable/TestCommitMarkableResourceBase.java

```diff
@@ -49,7 +50,7 @@ public final void setup() throws Exception {
 
 		File file = new File(System.getProperty("user.dir") + "/ObjectStore");
 		if (file.exists()) {
-			Utils.removeRecursive(file.toPath());
+			FileUtils.deleteRecursive(file.getAbsolutePath(), false);
```

Comment:
What was the problem with `Utils.removeRecursive`? Is it using features that aren't in 1.6?

Reply:
Yes. It's from 1.7 in Java.

Reply:
So do you mean that the code inside FileUtils.deleteRecursive() is using features that are not available 1.6

Reply:
Sorry, I probably explained wrongly.
The `FileUtils` uses the code which is available in 1.6. That's the reason why I took that class it into the changeset.
The `Utils` class uses the file nio utils methods which are not available on JDK6. That's the reason for replacing it.

Reply:
Ah right, got you thanks. I'll mark this conversation as resolved then.